autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -181,6 +181,11 @@ def update_execution(status: ExecutionStatus) -> ExecutionResult:
         credentials, creds_lock = creds_manager.acquire(user_id, credentials_meta.id)
         extra_exec_kwargs[field_name] = credentials
 
+    # Inject extra execution arguments for the blocks via kwargs
+    extra_exec_kwargs["user_id"] = user_id
+    extra_exec_kwargs["run_id"] = graph_exec_id
+    extra_exec_kwargs["graph_id"] = graph_id
```

Comment:
It's much clearer to have it as an initial value, also let's keep the graph_exec_id as graph_exec_id, since we also have node_exec_id:
https://github.com/Significant-Gravitas/AutoGPT/pull/9320/files#diff-1b278ebf10a9da0fb5030010222b3a6df2b05a5463cad428cd6c38a1541b0f73R173-R178

Reply:
done